Doc/library/warnings.rst

```diff
@@ -336,9 +336,9 @@ check::
         assert issubclass(w[-1].category, DeprecationWarning)
         assert "deprecated" in str(w[-1].message)
 
-One can also cause all warnings to be exceptions by using ``error`` instead of
-``always``. One thing to be aware of is that if a warning has already been
-raised because of a ``once``/``default`` rule, then no matter what filters are
+One can also cause all warnings to be exceptions by using ``"error"`` instead of
```

Comment:
I wouldn’t add the quotes here.  If I apply the doc here for my command-line parameters for example, I would write `python -W something::error`.  The quotes are only needed when using the Python API; I don’t think there is ambiguity as they are not documented as constants or enums.

Reply:
Thanks!  I've removed them.